doc/source/orbit.rst

```diff
@@ -346,6 +346,52 @@ behavior
 
 .. image:: images/lp-orbit-integration-EzJz.png
 
+.. _fastchar:
+
+Fast orbit characterization
+-----------------------
```

Comment:
My sphinx build complains that the --- aren't long enough (they need to extend at least as far as the title)